scripts/zones/Phanauet_Channel/npcs/Roi.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Phanauet Channel
+--  NPC: Roi
+-- Type: Standard NPC
+-- !pos -6.04 -2.75 13.833 1
+-- Rotation 193
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
```

Comment:
Are you able to add in him at least saying "..." in to the DefaultActions.lua file for this zone? Looks like the ID for that is `7228`

Also looking at the picture of the NPC on ffxiclopedia, it looks like he should be on the other side of the boat in the other white box on the ground? idk